addon/mixins/base-query-manager.js

```diff
@@ -4,8 +4,8 @@ import { computed } from '@ember/object';
 
 export default Mixin.create({
   apolloService: service('apollo'),
-
-  apollo: computed(function() {
-    return this.get('apolloService').createQueryManager();
-  }),
+  init() {
+    this._super(...arguments);
+    this.apollo = this.get('apolloService').createQueryManager();
```

Comment:
should this use a setter, i.e. `this.set("apollo", ...)` ?

Reply:
Yep, for consistency I should, although in an init hook you can do this safely.